### PR TITLE
Edit AboutUs and PPP

### PR DESCRIPTION
**Pull request by Ruizhi2001**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -31,9 +31,10 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ### Johnny Doe
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/ruizhi.png" width="200px">
```
Comment (samuelmui8) [inline docs/AboutUs.md:34]: Nice photo

**Review by samuelmui8 — COMMENTED** [review]
LGTM

**Review by teoks0199 — APPROVED** [review]
LGTM

**Review by samuelmui8 — APPROVED** [review]
LGTM

**Pull request merged**